### PR TITLE
feat: [FFM-5217]: Updating Javascript SDK to One Click Example

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

Removing the multiple code snippets and adding in a one click code example, for onboarding the Javascript SDK

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -658,116 +658,81 @@ onboarding:
...
+      <!DOCTYPE html>
+      <html lang="en-us">
+        <head>
+          <meta charset="utf-8" />
+          <meta http-equiv="Content-Language" content="en" />
+          <title>Harness Feature Flags Sample</title>
 
-      # Getting value for a particular feature flag
+          <!-- Use Harness SDK -->
+          <script src="https://unpkg.com/@harnessio/ff-javascript-client-sdk@1.4.8/dist/sdk.client-iife.js"></script>
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:672]: That version is WAY outa date

Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:672]: We're better off using the module approach.

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -658,116 +658,81 @@ onboarding:
...
-      # Cleaning up
-
-      Remove a listener of an event by cf.off.
-
-      ```
-      cf.off(Event.ERROR, () => {
-        // Do something when an error occurs
-      })
-      ```
+            let searchParams = new URLSearchParams(window.location.search)
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:680]: This code style is super old-skool

File: src/modules/75-cf/strings/strings.en.yaml [hunk 3]
```diff
@@ -658,116 +658,81 @@ onboarding:
...
-      <script>
-        var initialize = HarnessFFSDK.initialize
-        var Event = HarnessFFSDK.Event
-      </script>
-      ```
-
-      Remember to change the version 1.3.7 in the unpkg url accordingly.
+            // Replace placeholder with your SDK Secret to initialise your Harness Feature Flag
+            var cf = initialize(
+              'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:695]: does it work if you replace `'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'` with `'{{apiKey}}'` so it'll grab the generated api key so users don't have to manually replace it

File: src/modules/75-cf/strings/strings.en.yaml [hunk 4]
```diff
@@ -658,116 +658,81 @@ onboarding:
...
-      Remember to change the version 1.3.7 in the unpkg url accordingly.
+            // Replace placeholder with your SDK Secret to initialise your Harness Feature Flag
+            var cf = initialize(
+              'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 
+              {
+                identifier: identifier,
+              }
+            )
+
+            //Listen to events from the cf instance
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:701]:
```suggestion
            // Listen to events from the cf instance
```

File: src/modules/75-cf/strings/strings.en.yaml [hunk 5]
```diff
@@ -658,116 +658,81 @@ onboarding:
...
+            cf.on(Event.ERROR, () => {
+              log('Error with flag')
+              log(JSON.stringify(flagInfo, null, 2))
+            })
+          </script>
+      </html>
+      ```
+      # Test your application 
+
+      Run your application by opening `index.html` in your browser.
```
[the remark this answers is not in the document]
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:735]: I was going off docs, could you suggest a way to do it? As it is like this in other sites examples

File: src/modules/75-cf/strings/strings.en.yaml (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:672]: what is the more up to date version?

File: src/modules/75-cf/strings/strings.en.yaml (hunk 4 above)
Comment (crystal-fong) [reply src/modules/75-cf/strings/strings.en.yaml:701]: just for consistency

Comment (crystal-fong) [reply src/modules/75-cf/strings/strings.en.yaml:701]: also should it be `FF instance`?

Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: It now says 'List for events' :)

File: src/modules/75-cf/strings/strings.en.yaml (hunk 3 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: thanks - will change :)

**Review by crystal-fong — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
I don't like this.

I flip'n love it!

**Pull request merged**